src/main/scala/rps/GameRepository.scala

```diff
@@ -12,15 +17,54 @@ trait GameRepository {
 
 class InMemoryGameRepository extends GameRepository {
   private val map = TrieMap.empty[UserId, Play]
+  private val db = Database.forConfig("h2mem1") // TODO: we should db.close this!
   private var nextInt = new AtomicInteger
 
+  val plays = TableQuery[Plays]
+  val setupAction = plays.schema.create
+  val setupFuture = db.run(setupAction)
+  Await.result(setupFuture, Duration.Inf)
+
   override def save(play: Play): UserId = {
```

Comment:
you should always return a Future untill you really need the value, in order to avoid blocking situation